src/interface/components/VLIW/modal/VLIWLoadModalComponent.tsx

```diff
@@ -24,10 +22,9 @@ SF	F1 (R3)
 ADDI 	R2 R2 #1
 ADDI	R3 R3 #1
 BNE	R2 R5 LOOP
-`.trim();
+`;
```

Comment:
Hmm, so, were the `trim()` calls useful, then? I placed them to avoid placing the first line in the same line as the const definition, since it looks ugly.

Reply:
yes, they were for removing the first \n, should I revert then?